hca2scea-backend/hca2scea.py

```diff
@@ -202,6 +225,24 @@ def get_new_protocol_column_names(sdrf, counter):
 
     return new_column_names, counter
 
+def order_protocols(protocols_sdrf_before_sequencing):
+    collection_protocol_cols = []
+    dissociation_protocol_cols = []
+    enrichment_protocol_cols = []
+    library_preparation_protocol_cols = []
+    for col in list(protocols_sdrf_before_sequencing.columns):
+        if col.split(".protocol_core.protocol_id")[0] == "collection_protocol":
```

Comment:
No need to repeat `col.split(".protocol_core.protocol_id")[0]` 4 times.
Define a variable and use it in the 4 locations.